### PR TITLE
Added img class and global css

### LINKED ISSUE #102: Create a new class for the img and replace it with the global css in the styles.css file
Create a new class for the img and replace it with the global css in the styles.css file

### PR DESCRIPTION
**Pull request by bhavy2202**
## Purpose

The purpose of this Pull Request is to fix #102 

### Preview Link

https://pr-{PR_NUMBER}-CALCULATOR-WEBSITE.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,7 +78,8 @@
     <title>CALCULATOR | YOSHITHA</title>
   </head>
 
-  <body class="bg-img">
+  <body class="img">
```
Comment (YoshithaRathnayake) [inline index.html:81]: This is not for the background image. There is a image without a class add a class for that @bhavy2202

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by bhavy2202** [comment]
Please check now @YoshithaRathnayake

File: index.html [hunk 2]
```diff
@@ -69,6 +69,7 @@
     <link
       rel="icon"
       type="image/png"
+      class = "img"
```
Comment (YoshithaRathnayake) [inline index.html:72]: Remove it from this bcs its the favicon @bhavy2202

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -78,7 +79,8 @@
     <title>CALCULATOR | YOSHITHA</title>
   </head>
 
-  <body class="bg-img">
+  <body class="bg">
```
Comment (YoshithaRathnayake) [inline index.html:82]: Make this as bg-img

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -78,7 +79,8 @@
...
+    <img class="img" href="assets/img/Background Image.jpg" alt="">
```
Comment (YoshithaRathnayake) [inline index.html:83]: Remove this img bcs the image is inserted in styles.css file @bhavy2202

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/styles.css [hunk 5]
```diff
@@ -1,7 +1,7 @@
 /* Edo Font */
 @import url("http://fonts.cdnfonts.com/css/edo");
 
-.bg-img {
+.img {
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:4]: Remove this there is a css property called img {} make it as .img @bhavy2202

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: assets/css/styles.css (hunk 5 above)
Comment (bhavy2202) [reply assets/css/styles.css:4]: Should I change .img to img?

Comment (YoshithaRathnayake) [reply assets/css/styles.css:4]: no img to .img in the bottom not this @bhavy2202

File: index.html (hunk 4 above)
Comment (YoshithaRathnayake) [reply index.html:82]: Solve this @bhavy2202

File: index.html (hunk 3 above)
Comment (YoshithaRathnayake) [reply index.html:81]: Solve this @bhavy2202

**Comment by YoshithaRathnayake** [comment]
Now I am going to sleep I can't do anything bcs for our country its 11.00 pm and I am a children and I have to go to school tomorrow. @bhavy2202

**Comment by bhavy2202** [comment]
it's last day of hacktober fest, please I'm requesting you.

**Comment by YoshithaRathnayake** [comment]
> it's last day of hacktober fest, please I'm requesting you.

You could have send it little bit earlier

**Review by YoshithaRathnayake — APPROVED** [review]
Approved, Merged!

**Pull request merged**